java/ql/src/Frameworks/JavaEE/EJB/EjbSetSocketOrUrlFactory.ql

```diff
@@ -4,7 +4,6 @@
  *              or the stream handler factory used by URL. Such operations could
  *              compromise security or interfere with the EJB container's operation.
  * @kind problem
- * @problem.severity error
  * @security-severity 5.8
```

Comment:
I think the intention here was to remove the line below instead.

Reply:
Yes it was 👍